src/Interfaces/Options/Particles/IOptionsParticlesSizeAnimation.ts

```diff
@@ -0,0 +1,6 @@
+export interface IOptionsParticlesSizeAnimation {
+    enable: boolean;
+    size_min: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'size_min' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)